### PR TITLE
Updates to the post gallery.

### PR DESCRIPTION
**Pull request by weerd**
### What does this PR do?
This PR updates the styles for the RB posts in the gallery, wrapping them in `Card` components. It also adds the section header and styles for the "User Gallery".

image

### Any background context you want to provide?
I did a bit of reorganizing of the `Gallery` component and its corresponding sub-components.

### What are the relevant tickets/cards?
Refs https://www.pivotaltracker.com/story/show/151058515

### CODE REVIEW

File: resources/assets/components/ActionPage/ActionStepsWrapper.js [hunk 1]
```diff
@@ -5,9 +5,10 @@ import ActionStep from './ActionStep';
 import Revealer from '../Revealer';
 import { makeHash } from '../../helpers';
 import { Flex, FlexCell } from '../Flex';
+import { PostGalleryContainer } from '../Gallery/PostGallery';
 import { ReportbackUploaderContainer } from '../ReportbackUploader';
 import CompetitionContainer from '../../containers/CompetitionContainer';
-import { PostGalleryContainer, SubmissionGalleryContainer } from '../Gallery';
+import { SubmissionGalleryContainer } from '../Gallery/SubmissionGallery';
```
Comment (weerd) [inline resources/assets/components/ActionPage/ActionStepsWrapper.js]: Ran into this issue again with having to specify the component within the directory further down, even though they are getting exported from their specific directory into the Gallery directory and the `Gallery/index.js` file properly.

File: resources/assets/components/ActionPage/ActionStepsWrapper.js [hunk 2]
```diff
@@ -20,7 +21,7 @@ const ActionStepsWrapper = (props) => {
   );
 
   const postGallery = (
-    <PostGalleryContainer key="post_gallery" />
+    <PostGalleryContainer key="post_gallery" type="reportback" />
```
Comment (weerd) [inline resources/assets/components/ActionPage/ActionStepsWrapper.js]: Added a `type` for future use 🔮

File: resources/assets/components/ActionPage/ActionStepsWrapper.js (hunk 1 above)
Comment (itsjoekent) [reply resources/assets/components/ActionPage/ActionStepsWrapper.js]: Agghhhhh. Okay, maybe I should set aside time to look at this, and make an issue over at the Babel repo to get some Expert ™️ help

File: resources/assets/components/Gallery/index.js [hunk 3]
```diff
@@ -1,9 +1,7 @@
-export PostGallery from './PostGallery';
+export { PostGallery } from './PostGallery';
```
Comment (itsjoekent) [inline resources/assets/components/Gallery/index.js]: I don't think this is correct syntax?

Comment (weerd) [reply resources/assets/components/Gallery/index.js]: very possible 😬

Comment (weerd) [reply resources/assets/components/Gallery/index.js]:
```
export { name1, name2, …, nameN } from …;
```
Not the same? From [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export)

Comment (itsjoekent) [reply resources/assets/components/Gallery/index.js]: The export is fine but I assumed `./PostGallery` was only exporting a default component, in which case you shouldn't have to use the brackets

Comment (weerd) [reply resources/assets/components/Gallery/index.js]: oh very true! These imports/exports keep messing me up 😢

**Pull request merged**